build/pages/about/Officers-USC.html

```diff
@@ -0,0 +1,167 @@
+                    </div>
+                    <div class="header__search">
+                        <a href="/search" data-js="search_btn">
+                            <p class="sr_only">Search</p>
+                            <img alt="Search" src="/build/assets/icons/magnifier.svg" />
+                        </a>
+                    </div>
+                </div>
+            </div>
+        </header>
+        <div class="container container--header_accord">
+            <div class="header_accord" data-js="header_accord">
+                <menu class="link_list link_list--info" data-js="info_menu">
+                    <h1 class="link_list__hdng">Information:</h1>
+                    <li><a href="/announcements">Announcements</a></li>
+                    <li><a href="/events">Events</a></li>
+                    <li><a href="/organizations">Organizations</a></li>
+                    <li><a href="/usc">About the USC</a></li>
+                    <li><a href="/usc/officers">Officers of the USC</a></li>
+                </menu>
+                <menu class="link_list link_list--srvcs" data-js="srvcs_menu">
+                    <h1 class="link_list__hdng">Services:</h1>
+                    <li><a href="/services/volunteer">Volunteer Opportunities</a></li>
+                    <li><a href="/services/education">Education and Training</a></li>
+                    <li><a href="/usc/contact">Contact the USC</a></li>
+                    <li><a href="/usc-cite/contact">Contact the USC-CITE</a></li>
+                </menu>
+                <form class="search_form" action="/search" method="post" data-js="search_form">
+                    <label for="search" class="search_form__label">Search the USC Infosite</label>
+                    <input id="search" class="search_form__input" type="text" />
+                    <button class="search_form__btn" type="submit">
+                        <img alt="" src="/build/assets/icons/magnifier.svg" /><span class="sr_only">Search the USC Infosite</span>
+                    </button>
+                </form>
+            </div>
+        </div>
+
+        <main class="main">
+
+            <h1 class="main__heading">Officers of the University Student Council</h1>
+
+            <div class="main__container main__container--USC-officers">
+                <h2 class="main__sub-heading">USC OFFICERS</h2>
+                <span class="main__emphasis">AY 2023 - 2024</span>
+
+                <div class="main__USC-officers">
+                    <div class="USC-officers__chairpersons">
+                        <div class="main__container-card">
+                            <img src="/build/assets/images/USC-vice-chairperson-img.png" alt="" class="container-card__officer-image">
+                            <p class="container-card__officer-name">Hershey Geline Sumpay</p>
+                            <p class="container-card__officer-position">Chairperson</p>
+                        </div>
```

Comment:
There's an appropriate element for containing reusable content: `<article>`. [You can read more about it on MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article).

I'll retain what you built though. No need to change it for now.